### PR TITLE
NMS-14884: Replace PASSWORD-CLEAR with PASSWORD_CLEAR

### PR DESCRIPTION
**Pull request by mmahacek**
PR 2 of 2.  Replace references of `PASSWORD-CLEAR` with `PASSWORD_CLEAR` in docs and default configs.

### All Contributors

* [X] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [X] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### External References

* Jira (Issue Tracker): https://opennms.atlassian.net/browse/NMS-14884

### CODE REVIEW

**Review by RangerRick — CHANGES_REQUESTED** [review]
mostly looks good but it would be nice if you didn't rewrite the lines that are unchanged, only adding a space before the `/>`, it adds a lot of unnecessary churn

**Comment by mmahacek** [comment]
> mostly looks good but it would be nice if you didn't rewrite the lines that are unchanged, only adding a space before the `/>`, it adds a lot of unnecessary churn

Ah.  Most of that was my editor attempting to "clean up" the formatting. I'll revert those spaces.

**Comment by mmahacek** [comment]
Eh.  It changed too much.  Fixing.

File: opennms-base-assembly/src/main/filtered/etc/poller-configuration.xml [hunk 1]
```diff
@@ -268,9 +268,9 @@
          <parameter key="timeout" value="3000"/>
       </service>
       <service name="ActiveMQ" interval="300000" user-defined="false" status="on">
-         <parameter key="retry" value="1" />
-         <parameter key="timeout" value="3000" />
-         <parameter key="brokerURL" value="tcp://localhost:61616" />
+         <parameter key="retry" value="1"/>
+         <parameter key="timeout" value="3000"/>
+         <parameter key="brokerURL" value="tcp://localhost:61616"/>
```
Comment (mmahacek) [inline opennms-base-assembly/src/main/filtered/etc/poller-configuration.xml:273]:
```suggestion
         <parameter key="retry" value="1" />
         <parameter key="timeout" value="3000" />
         <parameter key="brokerURL" value="tcp://localhost:61616" />
```

File: opennms-base-assembly/src/main/filtered/etc/poller-configuration.xml [hunk 2]
```diff
@@ -403,6 +403,6 @@
    <monitor service="VMwareCim-HostSystem" class-name="org.opennms.netmgt.poller.monitors.VmwareCimMonitor"/>
    <monitor service="VMware-ManagedEntity" class-name="org.opennms.netmgt.poller.monitors.VmwareMonitor"/>
    <monitor service="MS-RDP" class-name="org.opennms.netmgt.poller.monitors.TcpMonitor"/>
-   <monitor service="ActiveMQ" class-name="org.opennms.netmgt.poller.monitors.ActiveMQMonitor" />
-   <monitor service="DeviceConfig" class-name="org.opennms.features.deviceconfig.monitors.DeviceConfigMonitor" />
+   <monitor service="ActiveMQ" class-name="org.opennms.netmgt.poller.monitors.ActiveMQMonitor"/>
+   <monitor service="DeviceConfig" class-name="org.opennms.features.deviceconfig.monitors.DeviceConfigMonitor"/>
```
Comment (mmahacek) [inline opennms-base-assembly/src/main/filtered/etc/poller-configuration.xml:407]:
```suggestion
   <monitor service="ActiveMQ" class-name="org.opennms.netmgt.poller.monitors.ActiveMQMonitor" />
   <monitor service="DeviceConfig" class-name="org.opennms.features.deviceconfig.monitors.DeviceConfigMonitor" />
```

**Comment by mmahacek** [comment]
That should be cleaner.  I left in a couple whitespace changes to fix a couple indentation inconsistencies.

**Review by RangerRick — APPROVED** [review]

**Pull request merged**